luigi2.h

```diff
@@ -3051,7 +3051,7 @@ int _UITableMessage(UIElement *element, UIMessage message, int di, void *dp) {
 			UIDrawControl(painter, row, UI_DRAW_CONTROL_TABLE_ROW | rowFlags, NULL, 0, 0, element->window->scale);
 
 			UIRectangle cell = row;
-			cell.l += UI_SIZE_TABLE_COLUMN_GAP * table->e.window->scale;
+			cell.l += UI_SIZE_TABLE_COLUMN_GAP - (table->hScroll ? (int64_t) table->hScroll->position : 0) * table->e.window->scale;
```

Comment:
`UI_SIZE_TABLE_COLUMN_GAP` should be multiplied by the scale, not the scroll position.

Reply:
this is needed to scroll the different entries horizontally, without this only the header is scrolled.